packages/expo/src/config.js

```diff
@@ -30,6 +31,11 @@ module.exports = {
       if (IS_PRODUCTION_MODE) return 'local-prod'
       return 'local-dev'
     }
+  },
+  codeBundleId: {
+    defaultValue: () => undefined,
```

Comment:
If this is supposed to be set the update ID, can we do that automatically?

You can try/catch around `require` to avoid crashing if a module isn't installed, but I'm not sure if this works in an Expo app:

```suggestion
    defaultValue: () => {
      try {
        const Updates = require('expo-updates')
        
        return Updates.updateId
      } catch {
        return undefined
      }
    },
```

Reply:
Discussed in Slack: we don't support automatic source map uploading with EAS update, so there's a danger that the `codeBundleId` would get out of sync by doing this and therefore break source map matching

We'll look at setting it automatically when we can also upload source maps automatically